go.mod

```diff
@@ -20,8 +20,8 @@ require (
 	github.com/c4milo/gotoolkit v0.0.0-20190525173301-67483a18c17a // indirect
 	github.com/clarketm/json v1.14.1
 	github.com/containers/image v3.0.2+incompatible
-	github.com/coreos/ignition/v2 v2.3.0
-	github.com/dmacvicar/terraform-provider-libvirt v0.6.2
+	github.com/coreos/ignition/v2 v2.9.0
+	github.com/dmacvicar/terraform-provider-libvirt v0.6.4-0.20201216193629-2b60d7626ff8
```

Comment:
Why does the libvirt terraform provider need to be updated?

Reply:
thanks! that probably slipped through the backport